### DISCUSSION TITLE
Track books / number of books read

### DISCUSSION THREAD

**Discussion by cvocvo** [babybuddy/babybuddy#666] [Ideas]
Could be just like a note but with the added benefit we could see numbers of books read all time, per year/month/week.

**Comment by cdubz** [comment]
I'm a tough sell on adding new core models. Notes + tags should get the job done for you here. The idea with tags is that Baby Buddy will at some point get to a future state where users can get more tag-based data in analytical formats like graphs.

E.g. if you add notes with a `book` tag and then have the ability to plot all entries with `book` tags over time you could essentially get to what you are thinking. We're not there yet but this does add to the use case.

See also https://github.com/babybuddy/babybuddy/issues/630#issuecomment-1581371858.

**Comment by cvocvo** [comment]
Agree on avoiding adding more core models if possible. I was thinking some kind of enum-based dropdown on notes for standardized types would be ideal. However, that seems like what you're accomplishing in a bit more free-form with tags. Maybe tags could just have some kind of option to autofill with previously used tags?
Agree on the plotting as well; I think that's a neat idea. My MVP is simpler with just a 'count' being sufficient. (The 1000 books before kindergarten initiative got me thinking about this: https://www.amesfreelibrary.org/childrens-slider/2020/04/03/1000-books-kindergarten)

I think my goal could also be accomplished if I could get a count of notes with tags out of this HomeAssistant integration: https://github.com/jcgoette/baby_buddy_homeassistant
Then I could store/graph the data there as well. I can ask them if that's feasible too.

**Reply by cdubz** [reply]
Yeah, getting that data from the API is definitely possible so I imagine it _should_ be possible with the HA integration with some work. I'm sure they'd be happy to discuss if you open a discussion/issue over there.

**Reply by cvocvo** [reply]
Cross-posted to here to facilitate that discussion: https://github.com/jcgoette/baby_buddy_homeassistant/issues/114
